packages/debian/metalk8s-sosreport-deb-2.4.0/debian/copyright

```diff
@@ -0,0 +1,9 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: metalk8s-sosreport
+Source: <url://example.com>
+#
+# Please double check copyright with the licensecheck(1) command.
```

Comment:
I guess we will want to use the Apache License (which is what we use for the project).